docs/developer_guide/secrets.md

```diff
@@ -0,0 +1,132 @@
+# Handling project secrets
+
+Arnold uses OpenShift's `Secret` objects to securely inject services credentials
+to running pods. Those `Secret`s are generated from a YAML file stored in:
```

Comment:
not really generated... set? taken?

Reply:
I think we can write generated as the Secrets are created from a template (`env_vars.yml.j2`) and encrypted variables definitions (`credentials.vault.yml`). WDYT?

Reply:
Yes. Missed that. Maybe add a paragraph or fast schema to explain that?

Reply:
I've refactored a bit the first paragraph.